.github/workflows/kickoff_featuretools_evalml_unit_tests.yaml

```diff
@@ -13,10 +13,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Run workflow for Featuretools unit tests
-        run: gh workflow run "Unit Tests - Featuretools with Woodwork main branch" --repo "alteryx/featuretools"
+        run: gh workflow run tests_with_latest_deps.yaml --repo "alteryx/featuretools"
```

Comment:
Should this be `tests_with_woodwork_main_branch.yaml`?

Reply:
Fixed